### DISCUSSION TITLE
Add option to disable being kicked for "out of order chat"

### DISCUSSION THREAD

**Discussion by Pantera07** [PurpurMC/Purpur#1038] [__Completed__] [answered]
Add option to disable being kicked for "out of order chat"
A lot of players are being kicked out by 'multiplayer.disconnect.out_of_order_chat' on my 1.19 server.

https://github.com/PaperMC/Paper/issues/8019
https://github.com/PaperMC/Velocity/issues/756

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in build 1690 https://github.com/PurpurMC/Purpur/commit/6d9b2a1a13833a5512cbd414df31f6df85f7cd61
